gradle.properties

```diff
@@ -15,6 +15,22 @@
 # limitations under the License.
 #
 
+group=org.apache.spark.kubernetes.operator
+version=0.1.0
+
+fabric8Version=6.12.1
+commonsLang3Version=3.14.0
+commonsIOVersion=2.16.1
```

Comment:
fabric8 client, commons library & log4j versions are designed to be inline with Apache Spark dependency version.